frontend/src/pages/MissionView/MissionView.tsx

```diff
@@ -0,0 +1,106 @@
+import { generatePath, useNavigate, useParams } from "react-router-dom";
+import Button, { BUTTON_VARIANT } from "../../components/@common/Button/Button";
+import Container from "../../components/@common/Container/Container";
+import useTokenContext from "../../hooks/useTokenContext";
+import styles from "./MissionView.module.css";
+import { PARAM, PATH } from "../../constants/path";
+import { fetchMissionRequirements } from "../../api";
+import { useEffect, useState } from "react";
+import highlighter from "highlight.js";
+import "github-markdown-css/github-markdown-light.css";
+import "highlight.js/styles/github.css";
+import { Mission } from "../../../types/domains/recruitments";
+import { getMissionLabel } from "../../hooks/useMission";
+import { MISSION_STATUS } from "../../constants/recruitment";
+import { AxiosError } from "axios";
+
+const MissionView = () => {
+  const navigate = useNavigate();
+
+  const { recruitmentId, missionId } = useParams<{ recruitmentId: string; missionId: string }>();
+  const [mission, setMission] = useState<Mission | null>(null);
+  const description = mission?.description ?? "";
+
+  const submittable = !mission || mission.submittable;
+  const missionLabel = getMissionLabel(
+    mission?.submitted ?? false,
+    mission?.status ?? MISSION_STATUS.UNSUBMITTABLE
+  );
+
+  if (!recruitmentId || !missionId) {
+    throw new Error("recruitmentId 또는 missionId가 없습니다.");
+  }
+
+  const { token } = useTokenContext();
+
+  const goBack = () => {
+    navigate(-1);
```

Comment:
명시적으로 `내 지원 현황` 페이지인 `/applications/me` 경로로 라우팅하는 것이 더 적절할까요?

<img width="813" alt="image" src="https://github.com/user-attachments/assets/1c1f9117-32df-4f46-9095-952441283870">

```suggestion
    navigate('/applications/me'); // (예시)
```

Reply:
a:
- '뒤로 가기' 기능이 의도라면 현재가 더 적절하다고 생각합니다. 
- 내 지원현황 페이지로 명식적으로 이동시키는 것이 의도라면, 함수 이름도 그에 맞춰서 같이 바꾸는 게 더 혼동의 여지가 적다고 생각해요 :)

Reply:
이 의도에 대해서는 구체적으로 논의된 적이 없어서요!
일단 '뒤로 가기'기능으로 구동하더라도 큰 문제가 되지 않으니 적용해 보고 적절치 않으면 다음 이슈에서 진행해보는 것이 좋을 것 같아요 😁